### PR TITLE
Added alternative with A SKU for testing premium

### PR DESCRIPTION
**Pull request by marclelijveld**
Added new option for testing Power BI premium functionality with an A SKU, as described in other documentation around Power BI Premium.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@marclelijveld : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger-2** [comment]
@peter-myers : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
@peter-myers 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"

**Comment by peter-myers** [comment]
#sign-off

File: powerbi-docs/guidance/migrate-azure-analysis-services-to-powerbi-premium.md [hunk 1]
```diff
@@ -168,12 +168,13 @@ A Pro (or PPU) license is required to publish and manage Power BI content. Pro l
...
+AAS offers the **D** and **B** SKUs at lower cost with reduced service-level agreements and/or fewer features than the **S** SKUs. Some AAS customers use these SKUs for development and test environments. While there's no direct equivalent in Power BI, it might make sense to use PPU licenses for development and test environments. Such environments typically don't have large numbers of users because they're limited to developers and testers. Alternatively, consider using an **A** SKU in Azure for testing Premium capacity functionality.
 
 For more information, see:
 
 - [Power BI pricing](https://powerbi.microsoft.com/pricing/)
 - [Azure Analysis Services pricing](https://azure.microsoft.com/pricing/details/analysis-services/)
+- [Purchase A SKU for testing and other scenarios](https://docs.microsoft.com/en-us/power-bi/enterprise/service-admin-premium-testing?WT.mc_id=DP-MVP-5003435#purchase-a-skus-for-testing-and-other-scenarios)
```
Comment (Court72) [inline powerbi-docs/guidance/migrate-azure-analysis-services-to-powerbi-premium.md:177]:
```suggestion
- [Purchase A SKU for testing and other scenarios](https://docs.microsoft.com/power-bi/enterprise/service-admin-premium-testing?WT.mc_id=DP-MVP-5003435#purchase-a-skus-for-testing-and-other-scenarios)
```

**Pull request merged**